hwilib/commands.py

```diff
@@ -44,7 +45,7 @@ def enumerate(password=''):
 def find_device(device_path, password='', device_type=None, fingerprint=None):
     devices = enumerate(password)
     for d in devices:
-        if device_type is not None and d['type'] != device_type:
+        if device_type is not None and d['type'] != device_type and d['model'] != device_type:
```

Comment:
do we want to attempt opening it if `device_type` is `None`?

Reply:
I suspect this is supposed to be an `or`

Reply:
No, this is correct.

When `device_type` is `None`, we are trying to find the device via fingerprint. So if it is `None`, then we still continue and just try every device. If `device_type` is not `None`, then the user has specified a type and we need to filter on that. So we will abort if the specified type does not match both the type and the model given by `enumerate`. If one of them match, then we should not skip checking this device but rather continue.